packages/api/src/controllers/revenueTypeController.js

```diff
@@ -154,7 +155,10 @@ const revenueTypeController = {
       const trx = await transaction.start(Model.knex());
       const { revenue_type_id } = req.params;
       const farm_id = req.headers.farm_id;
-      const data = { revenue_name: req.body.revenue_name };
+      const data = {
+        revenue_name: req.body.revenue_name,
+        custom_description: req.body.custom_description ?? null,
```

Comment:
I need to change this to undefined so it passes tests and does not overwrite the description. 

But just figuring out why it is failing as is right now since null is valid.

Reply:
Classic forgot to migrate testing db goodbye 2 hours. 😢  Anyways this actual problem is fixed.